crates/biome_js_analyze/tests/specs/nursery/noMisleadingCharacterClass/invalid.js.snap

```diff
@@ -1148,6 +1161,219 @@ invalid.js:63:9 lint/nursery/noMisleadingCharacterClass ━━━━━━━━
     62 │ var r = new globalThis.RegExp("[🇯🇵]", "");
   > 63 │ var r = new globalThis.RegExp("[\u{1F468}\u{200D}\u{1F469}\u{200D}\u{1F466}]", "u");
        │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    64 │ var r = new window.RegExp("[❇�]", "");
+    65 │ var r = new window.RegExp("[👍]", "");
+  
+
+```
+
+```
+invalid.js:64:9 lint/nursery/noMisleadingCharacterClass ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Unexpected combined character in the character class.
+  
+    62 │ var r = new globalThis.RegExp("[🇯🇵]", "");
+    63 │ var r = new globalThis.RegExp("[\u{1F468}\u{200D}\u{1F469}\u{200D}\u{1F466}]", "u");
+  > 64 │ var r = new window.RegExp("[❇�]", "");
+       │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    65 │ var r = new window.RegExp("[👍]", "");
+    66 │ var r = new global.RegExp("[❇�]", "");
+  
+
+```
+
+```
+invalid.js:65:9 lint/nursery/noMisleadingCharacterClass  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Unexpected surrogate pair in character class. Use the 'u' flag.
+  
+    63 │ var r = new globalThis.RegExp("[\u{1F468}\u{200D}\u{1F469}\u{200D}\u{1F466}]", "u");
+    64 │ var r = new window.RegExp("[❇�]", "");
+  > 65 │ var r = new window.RegExp("[👍]", "");
+       │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    66 │ var r = new global.RegExp("[❇�]", "");
+    67 │ var r = new global.RegExp("[👍]", "");
+  
+  i Safe fix: Add unicode u flag to regex
+  
+    63 63 │   var r = new globalThis.RegExp("[\u{1F468}\u{200D}\u{1F469}\u{200D}\u{1F466}]", "u");
+    64 64 │   var r = new window.RegExp("[❇�]", "");
+    65    │ - var·r·=·new·window.RegExp("[👍]",·"");
+       65 │ + var·r·=·new·window.RegExp("[👍]","u");
```

Comment:
Auto fix remove the space after the comma. It would be better if this could be fixed.

Reply:
I'm currently using the auto-generated function `make::js_call_argument_list` to create a node that suggests as arguments. Is there a way to add spaces between the separator and the value?
```
make::js_call_argument_list([literal, s], Some(make::token(T![,])))
```

Reply:
You can call `with_trailing_trivia` on the token:
```
make::js_call_argument_list(
  [literal, s],
  Some(
    make::token(T![,]).with_trailing_trivia([(TriviaPieceKind::Whitespace, " ")])
  ),
)
```

Reply:
Awesome! Thanks.